_posts/2017-11-23-ABRT-in-container.markdown

```diff
@@ -0,0 +1,224 @@
+---
+layout: post
+title:  "ABRT in openshift container"
+date:   2017-11-24 12:18:21 +0100
+author: Julius Milan <jmilan@redhat.com>
+visible: 1
+categories: abrt openshift container deploy
+---
+ABRT is able to run inside an openshift resp. docker container and catch certain subset of
+problems occured on the host on which the container is running including all other containers
```

Comment:
s/problems occured/problems that occurred/
s/on which the container/where the container/